test/controllers/StartDeclarationControllerSpec.scala

```diff
@@ -74,6 +87,26 @@ class StartDeclarationControllerSpec extends SpecBase with BeforeAndAfterEach {
       redirectLocation(result).value mustEqual controllers.routes.SessionExpiredController.onPageLoad().url
       verify(mockUserAnswerRepository, times(1)).get(any())
     }
-
+  }
+  "redirect to Tasklist for a GET if SectionPage is not completed" in {
+
+    when(mockUserAnswerRepository.get(any())).thenReturn(Future.successful(Some(emptyUserAnswers
+      .set(Section1Page, false)
+      .flatMap(_.set(Section2Page, false))
```

Comment:
No need to set all Section page values here... just set one for true and one for false